assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -191,19 +229,9 @@ function Carousel() {
   );
 
   useLayoutEffect(() => {
-    if (hasHorizontalOverflow) {
-      const currentPageRef = pageRefs.current[currentPageId];
-
-      if (!currentPageRef || !currentPageRef.scrollIntoView) {
-        return;
-      }
-
-      currentPageRef.scrollIntoView({
```

Comment:
Where is this code now? Looks like it's still needed...

Reply:
Reverted